java/java.source.base/src/org/netbeans/api/java/source/TreeMaker.java

```diff
@@ -1042,7 +1042,18 @@ public ReturnTree Return(ExpressionTree expression) {
     public SwitchTree Switch(ExpressionTree expression, List<? extends CaseTree> cases) {
         return delegate.Switch(expression, cases);
     }
-    
+
+    /**
+     * Creates a new SwitchExpressionTree.
+     *
+     * @param expression the expression which provides the value to be switched.
+     * @param cases the list of cases, or an empty list.
+     * @see com.sun.source.tree.SwitchExpressionTree
```

Comment:
I am afraid this will need the usual spec version increment, @since, apichanges.xml entry, etc.

Reply:
Changes added

Reply:
changes added